src/Symfony/Component/Validator/Constraints/LengthValidator.php

```diff
@@ -30,7 +30,7 @@ public function validate($value, Constraint $constraint)
             throw new UnexpectedTypeException($constraint, Length::class);
         }
 
-        if (null === $value || ('' === $value && $constraint->allowEmptyString)) {
+        if (null === $value || '' === $value) {
```

Comment:
`allowEmptyString` defaults to false, which means we should not allow empty string anymore, isnt it?

Reply:
Yes, that's true. I am going to remove that.